.github/workflows/ci.yml

```diff
@@ -149,3 +154,27 @@ jobs:
 
       - name: Cargo nextest
         run: cargo nextest run --workspace
+
+ test_wasm:
+    name: Test wasm
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Install
+      run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
```

Comment:
I've done `cargo install wasm-pack` in the past; is there an advantage to doing it this way?

Reply:
Aah; one advantage of the script by the looks of it is that it'll download pre-compiled binaries if they exist; that's good to know!